src/tests/test_storage.py

```diff
@@ -0,0 +1,8 @@
+def setup_function(function):
+    ...
+
+def teardown_function(function):
+    ...
+
+def test_storage_disks_infos():
+    ...
```

Comment:
I don't really know how to test that. It would need simulating hard drives but the current implementation excludes virtual devices. Any idea?

Reply:
Don't know much what needs testing but I suppose we should look into mocking `pyudev`